src/core/File.ts

```diff
@@ -450,6 +450,8 @@ export class File {
             if (/\.js$/.test(this.info.fuseBoxPath) && this.context.fuse && this.context.fuse.producer) {
```

Comment:
I think it can be `jsx` too. Or other things that should be interpreted as assets

Reply:
Line 454 catches all else statements, so .jsx will be caught. I wonder if all of them should be throwing an error?

Reply:
Yeah, just jsx isn't an assets, it's a module ;-)

Reply:
Done. Does it make sense for one to throw warnings and the else to throw errors? I would think that errors should always be thrown is something is missing?